js/deer-record.js

```diff
@@ -12,6 +12,8 @@
 import { default as UTILS } from './deer-utils.js'
 import { default as config } from './deer-config.js'
 
+UTILS.worker = await import("./js/worker.js").then(module => {return module.default}).catch(err => {return {"error":err}})
```

Comment:
This can't import on UTILS? This will make a mess if we have to do it this way...